resources/sdk/pureclouddotnet-guest/templates/api_doc.mustache

```diff
@@ -0,0 +1,88 @@
+---
+title: {{classname}}
+---
+## {{packageName}}.Api.{{classname}}{{#description}}
+{{description}}{{/description}}
+
+All URIs are relative to *{{basePath}}*
+
+| Method | HTTP request | Description |
+| ------------- | ------------- | ------------- |
+{{#operations}}{{#operation}}| [**{{operationId}}**]({{classname}}.html#{{operationIdLowerCase}}) | **{{httpMethod}}** {{path}} | {{#summary}}{{summary}}{{/summary}} |
+{{/operation}}{{/operations}}{: class="table table-striped"}
+
+{{#operations}}
+{{#operation}}
+<a name="{{operationIdLowerCase}}"></a>
+
+## {{#returnType}}{{#returnTypeIsPrimitive}}**{{returnType}}**{{/returnTypeIsPrimitive}}{{^returnTypeIsPrimitive}}[**{{returnType}}**]({{returnBaseType}}.html){{/returnTypeIsPrimitive}}{{/returnType}}{{^returnType}}void{{/returnType}} {{operationId}} ({{#allParams}}{{{dataType}}} {{paramName}}{{^required}}{{#optionalMethodArgument}} = null{{/optionalMethodArgument}}{{/required}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
+
+{{#deprecated}}<span style="background-color: #f0ad4e;display: inline-block;padding: 7px;font-weight: bold;line-height: 1;color: #ffffff;text-align: center;white-space: nowrap;vertical-align: baseline;border-radius: .25em;margin: 10px 0;">DEPRECATED</span>{{/deprecated}}
+
+{{{summary}}}{{#notes}}
+
+{{{notes}}}{{/notes}}
+
+Requires {{permissionType}} permissions: 
+
+{{#permissions}}
+* {{.}}
+{{/permissions}}
+
+### Example
+```{"language":"csharp"}
+using System;
+using System.Diagnostics;
+using {{packageName}}.Api;
+using {{packageName}}.Client;
+using {{packageName}}.Model;
+
+namespace Example
+{
+    public class {{operationId}}Example
+    {
+        public void main()
+        { {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}
+            // Configure HTTP basic authorization: {{{name}}}
+            Configuration.Default.Username = 'YOUR_USERNAME';
+            Configuration.Default.Password = 'YOUR_PASSWORD';{{/isBasic}}{{#isApiKey}}
+            // Configure API key authorization: {{{name}}}
```

Comment:
Rewrite this  `isApiKey` block to use `Configuration.Default.SetJwt(chatInfo.Jwt);` with a comment above noting to get the JWT from creating a new chat conversation. Don't need the "uncomment below" parts.

Reply:
Have updated the documentation. Looks like the UI isn't showing it though because I did a force push.